kernel-commits.mk

```diff
@@ -1,8 +1,8 @@
-KERNEL_COMMIT_amd64_v5.10.186_generic = 37256d6aff33
-KERNEL_COMMIT_amd64_v6.1.38_generic = 2049bda2c9ed
-KERNEL_COMMIT_amd64_v6.1.38_rt = ef571b34f5ad
-KERNEL_COMMIT_amd64_v6.1.68_generic = 662de9abec4d
-KERNEL_COMMIT_arm64_v5.10.104_nvidia = 02c64aba1d9f
-KERNEL_COMMIT_arm64_v5.10.186_generic = 594f2361a83f
-KERNEL_COMMIT_arm64_v6.1.38_generic = aa4b24ade858
-KERNEL_COMMIT_riscv64_v6.1.38_generic = 72192e3cbc74
+KERNEL_COMMIT_amd64_v5.10.186_generic = d61682724485
```

Comment:
@jsfakian , I guess these changes of kernel commits are unintentional, right? Probably you need to rebase your branch...

Reply:
We have changed the eve-kernel but have not updated EVE. If you think it is better to keep only the change I did I will update my PR.

Reply:
Understood, I was confuse because you just mention in PR's description about the firmware, so it was not clear whether kernel update was required as well or not. Usually it's better to send kernel updates in a separated PR when they are not related/required by the changes in the PR. Also, it makes easy to solve any conflict while rebasing due to kernel updates, so I would prefer to have the kernel update commit into a separated PR if you don't mind...

Reply:
> We have changed the eve-kernel but still need to update EVE. If it is better to keep only my changes, I will update my PR. The other commits are spelling fixes, so it is fine to remove them.

preferably it must be done using ./tools/update_kernel_commits.py script